src/app/chat/views/message_format.nim

```diff
@@ -21,9 +21,12 @@ proc renderInline(self: ChatMessageList, elem: TextItem): string =
   of "code": result = fmt(" <code>{value}</code> ")
   of "emph": result = fmt(" <em>{value}</em> ")
   of "strong": result = fmt(" <strong>{value}</strong> ")
+  of "strong-emph": result = fmt(" <strong><em>{value}</em></strong> ")
+  of "del": result = fmt(" <span class=\"strike\">{value}</span> ")
```

Comment:
I added this a bit differently in the keyboard shortcuts PR. Maybe you could base your branch off that branch?

Reply:
Ah crap, I didn't notice. I'll rebase yeah. Your handling of `del` is simpler as well

Reply:
Updated